src/olympia/addons/serializers.py

```diff
@@ -383,8 +383,9 @@ def fake_object(self, data):
         )
 
         # Attach attributes that do not have the same name/format in ES.
-        obj.tag_list = data['tags']
-        obj.disabled_by_user = data['is_disabled']  # Not accurate, but enough.
+        obj.tag_list = data.get('tags', [])
+        obj.disabled_by_user = data.get(
+            'is_disabled', False)  # Not entirely accurate, but enough.
```

Comment:
Move the comment to the line above it and it starts to look way better again imho (no need to break the line)